src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  /* background-color: rgb(231, 228, 228); */
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+/* .nav {
+  margin-right: 50px;
+} */
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  white-space: nowrap;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:nth-child(1) {
+  margin-left: 0;
+}
+
+.nav__link.is-active {
+  color: #00acdc !important;
+}
+
+.nav__link.is-active::after {
```

Comment:
```suggestion
.is-active::after {
```

Reply:
<img width="979" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/67204e1c-f381-4370-9227-85b8c4300e1e">
Fix similar problem in all code